README.md

```diff
@@ -197,3 +197,9 @@ Sample code:
   ]}
 />
 ```
+
+## Credit
+
+<a href="https://www.netlify.com">
```

Comment:
This can also be done in Markdown with:

```md
[![Netlify](https://www.netlify.com/img/global/badges/netlify-color-accent.svg)](https://www.netlify.com)
```

Which renders as:

[![Netlify](https://www.netlify.com/img/global/badges/netlify-color-accent.svg)](https://www.netlify.com)